contrib/devtools/bitcoin-tidy/CMakeLists.txt

```diff
@@ -8,7 +8,11 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
 set(CMAKE_CXX_EXTENSIONS False)
 
-# TODO: Figure out how to avoid the terminfo check
+set(CMAKE_DISABLE_FIND_PACKAGE_CURL ON)
```

Comment:
What Clang version should I test?

Asking because the `find_package(CURL)` command is not used in my clang-14 installation.

Reply:
> What Clang version should I test?

I assume any recent version, or the same one that is used in the CI job that uses this tool (18).

Reply:
Right. I has been [added](https://github.com/llvm/llvm-project/commit/a3b0dde4edb93248f74d86c63b0ae108854004e6) since clang-17.0